test/TestAppUtils/Themes/Generic.xaml

```diff
@@ -44,11 +44,17 @@
                                         <SymbolIcon Symbol="Back"/>
                                     </Button>
                                     <Button x:Name="ToggleThemeButton"
-                                        VerticalAlignment="Stretch"
+                                        VerticalAlignment="Stretch" Margin="0,0,4,0"
                                         Style="{ThemeResource AccentButtonStyle}"
                                         AutomationProperties.AutomationId="__ToggleThemeButton" 
                                         Content="Toggle theme">
                                     </Button>
+                                    <Button
+                                        x:Name="SwitchCIDimensionsButton"
+                                        VerticalAlignment="Stretch" Margin="0,0,4,0"
+                                        Style="{ThemeResource AccentButtonStyle}"
+                                        AutomationProperties.AutomationId="__SwitchCIDimensions"
+                                        Content="Switch to CI dimension"/>
```

Comment:
> CI dimensio [](start = 58, length = 12)

nit: Can we make this more descriptive ? Switch to smaller size for lab or something like that ?

Reply:
Update the text